e2e/cypress/support/ui_commands.js

```diff
@@ -421,3 +421,111 @@ Cypress.Commands.add('fileUpload', (targetInput, fileName = 'mattermost-icon.png
         );
     });
 });
+
+/**
+ * Upload a file on target input in binary format -
+ * @param {String} Subject - cypress/fixtures/matterMost.tar.gz
+ * @param {String} filePath - cypress/fixtures/matterMost.tar.gz
+ * @Param {String} fileType - application/gzip
+ */
+Cypress.Commands.add('uploadFile2', {prevSubject: true}, (subject, fileName, fileType) => {
+    cy.log('Upload process started .FileName:' + fileName);
+    cy.readFile(fileName, 'binary').then((content) => {
+        return Cypress.Blob.binaryStringToBlob(content, fileType).then((blob) => {
+            const el = subject[0];
+            const testFile = new File([blob], fileName, {type: fileType});
+            const dataTransfer = new DataTransfer();
+            dataTransfer.items.add(testFile);
+            el.files = dataTransfer.files;
+            cy.wrap(subject).trigger('change', {force: true});
+        });
+    });
+});
+
+Cypress.Commands.add('inputFileUpload', (targetInput, fileName, mimeType) => {
```

Comment:
I don't see this command used elsewhere.

Reply:
While building the feature automation , we expose required common methods even if it doesn't gets used .Might be used in future.

If this is not required , will remove

Thanks for other valuable suggestions .i will check on those

Reply:
Yup, please remove if not use currently.

Reply:
UnUsed common commands are removed